packages/web-app/src/routes/document-service-proxy.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+
+const router = express.Router();
+
+const documentServiceProxyController = require('../controllers/document-service-proxy');
+
+router.get('/:appealId/:documentId', documentServiceProxyController.getDocument);
+
+module.exports = router;
```

Comment:
It doesn't appear that we have any unit tests for `routes/document-service-proxy.js`?